recipes/genome-properties/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: genome-properties
+  version: {{ version }}
+
+source:
+  url: https://github.com/SilasK/genome-properties/archive/v{{ version }}.tar.gz
```

Comment:
It seems like you are trying to contribute a fork of https://github.com/ebi-pf-team/genome-properties - it would be better if the bioconda recipe used the official upstream repository.

I see you have some [changes](https://github.com/ebi-pf-team/genome-properties/compare/master...SilasK:master) in your fork of `genome-properties` - if they were important to this bioconda recipe you should first submit it as a pull request back to the @ebi-pf-team maintainers. 

However looking at your ciommits there it seem to be mainly whitespace changes?

Reply:
The authors intend genome properties to be used as command line according to the [docs](https://genome-properties.readthedocs.io/en/latest/calculating.html). 
However, it's a pain to install the required Perl modules. If I managed to install it I push my changes to the official repository. Is there a way to adopt the default to the conda $PREFIX?